modules/augment/src/rotate.cpp

```diff
@@ -0,0 +1,40 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html.
+
+#include "precomp.hpp"
+namespace cv {
+namespace augment {
+
+    Rotate::Rotate(Range _angleRange)
+    {
+        this->angleRange = _angleRange;
+        this->resetRandom();
+    }
+
+    Rotate::Rotate() : Rotate::Rotate(Range(0, 180)) {}
+
+    void Rotate::resetRandom()
+    {
+        this->currentAngle = rng.uniform(this->angleRange.start, this->angleRange.end + 1);
+    }
```

Comment:
I we want to choose angle randomply that means an every call of `Rotate::image` select an angle every randomly time from the range.

Reply:
I used that in the start, but there was an issue with that.. the user needs to have consistent random values to apply it the same to the image and the ground truth data, before the angle can change again.

That will be only used in the lower level api.. the user will deal with Augmenter class which will handle all that for him.